app/_src/gateway/kong-enterprise/db-encryption.md

```diff
@@ -45,6 +45,35 @@ export KONG_KEYRING_RECOVERY_PUBLIC_KEY=/path/to/generated/cert.pem
 All nodes in the Kong cluster should share the same `keyring_enabled` and `keyring_strategy` configuration values.
 The generated private key `key.pem` is not required to configure Kong, or to start Kong. Please refer to the [disaster recovery](#disaster-recovery) section for more information on how to use private keys in keyring management.
 
+{% if_version gte:3.8.x %}
+#### (Optional): Configure license payload encryption
```

Comment:
```suggestion
#### Configure license payload encryption
```
Let's not have "optional" in the section header. Technically any section could be optional, especially since this isn't in step order.